CMakeLists.txt

```diff
@@ -94,6 +74,38 @@ add_subdirectory(platform)
 add_subdirectory(rtos)
 add_subdirectory(targets)
 
+#Verify that the selected application OS profile is supported by the Mbed target
+if(NOT MBED_CONF_RTOS_PRESENT)
+    if(NOT "bare-metal" IN_LIST MBED_TARGET_SUPPORTED_APPLICATION_PROFILES)
+        message(FATAL_ERROR
+            "Baremetal profile is not supported for this target")
+    else()
+        set(MBED_C_LIB "small" CACHE STRING "")
+        message(${MBED_C_LIB})
```

Comment:
is this debug print,not needed? or if we want to say something, it should be more detailed.